### PR TITLE
[SKILLONOMY] Hotfix release SKILLONOMY-149

### PR DESCRIPTION
**Pull request by dyudyunov**
**Description:** Added translations

**Youtrack:** https://youtrack.raccoongang.com/issue/SKILLONOMY-149

### CODE REVIEW

File: lms/envs/aws.py [hunk 1]
```diff
@@ -1038,3 +1038,6 @@
 EDEOS_API_SECRET = AUTH_TOKENS.get('EDEOS_API_SECRET', '')
 EDEOS_API_URL = AUTH_TOKENS.get('EDEOS_API_URL', '')
 # EDEOS
+
+# Number of courses per page setting (for home page)
+HOMEPAGE_COURSE_MAX = ENV_TOKENS.get('HOMEPAGE_COURSE_MAX', HOMEPAGE_COURSE_MAX)
```
Comment (cmltaWt0) [inline lms/envs/aws.py]: This change is not related to the issue.

Comment (dyudyunov) [reply lms/envs/aws.py]: fixed
